common/processes/platform/tiny_process_library/tiny_process_library.cpp

```diff
@@ -64,6 +66,19 @@ error::Error Process::Start(OutputCallback stdout_callback, OutputCallback stder
 		return MakeError(ProcessAlreadyStartedError, "Cannot start process");
 	}
 
+	// Tiny-process-library doesn't give a good error if the command isn't found (just returns
+	// exit code 1). If the path is absolute, it's pretty easy to check if it exists. This won't
+	// cover all errors (non-absolute or unset executable bit, for example), but helps a little,
+	// at least.
+	if (args_.size() > 0 && path::IsAbsolute(args_[0])) {
```

Comment:
Because relative paths are resolved by the shell by looking through `PATH`, which I can't do here without doing it manually. And I wanted to keep it simple. This is, as mentioned, just a best effort for a slightly better error message, and not a critical thing.